runtime/doc/various.txt

```diff
@@ -579,6 +579,10 @@ Acting on multiple lines behaves as follows:
   transformed to empty comments (e.g. `/**/`). Comment markers are aligned to
   the least indented line.
 
+Matching 'commentstring' does not account for whitespace in comment markers.
```

Comment:
```suggestion
Matching 'commentstring' does not account for whitespace in comment markers when uncommenting.
```

Reply:
Matching 'commentstring' is about deciding what action should be done: commenting or uncommenting. So I don't think it is a good idea to add this detail.